packages/material-ui-lab/src/internal/pickers/PickersToolbarButton.tsx

```diff
@@ -1,59 +1,46 @@
 import * as React from 'react';
-import clsx from 'clsx';
 import Button, { ButtonProps } from '@material-ui/core/Button';
-import { MuiStyles, WithStyles, withStyles } from '@material-ui/core/styles';
+import styled from '@material-ui/styled-engine';
```

Comment:
```suggestion
import { experimentalStyled as styled } from '@material-ui//core/styles';
```

Reply:
> I agree that we should have probably waited. The only thing I would change on the PR is the import to experimentalStyled, just for the sake of keeping it consistent with the rest of the components.

But these are not the same problems I want to solve here. I don't need `sx`. I don't need `classes`. I don't need transient props. I don't need overrides resolvers or any resolver for that matther. I don't need slots. I just want to style a component. That shouldn't come with all the cost just so that you can use the same import.

Reply:
But you do want default theme generally in the styles. For components that are accessing the theme, this will break. If you don't provide name and slot you won't have style overrides and overrides resolver. If you even don't want the `sx` prop, you can provide in the mui options `skipSx`.

```diff
-const PickersToolbarButtonRoot = styled(Button)({
+const PickersToolbarButtonRoot = styled(Button, {}, { skipSx: true })({
```

This is equivalent with `styled()` from `@material-ui/styled-engine`, it just adds the default theme, so that if the `ThemeProvider` is not used, it would not fail. It would be harder to follow later if some styles based on theme are added, why the component is crashing all of a sudden.